aea/protocols/signing/signing.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package fetch.aea.Signing;
+package fetchai.aea.fetchai.signing;
```

Comment:
Why the fetchai twice? Why not simply `aea.author_name.package_name`? I don't think aea should be tied to fetchai like this

Reply:
ok, I'm going to remove the first `fetchai`